docs/dev/table_api.md

```diff
@@ -1022,69 +1022,70 @@ Temporal intervals can be represented as number of months (`Types.INTERVAL_MONTH
 
 <div class="codetabs" markdown="1">
 <div data-lang="java" markdown="1">
 {% highlight java %}
 Table table = input
-  .window(GroupWindow w)  // define window
+  .window(window.as("w"))  // define window with alias
+  .groupBy("w") 
   .select("b.sum")        // aggregate
 {% endhighlight %}
 </div>
 
 <div data-lang="scala" markdown="1">
 {% highlight scala %}
 val table = input
-  .window(w: GroupWindow) // define window
+  .window(window as 'w) // define window with alias
+  .groupBy('w) 
   .select('b.sum)         // aggregate
 {% endhighlight %}
 </div>
 </div>
 
-In streaming environments, group-window aggregates can only be computed in parallel, if they are *keyed*, i.e., there is an additional `groupBy` attribute. Group-window aggregates without additional `groupBy`, such as in the example above, can only be evaluated in a single, non-parallel task. The following example shows how to define a keyed group-window aggregation on a table. 
+In streaming environments, window aggregates can only be computed in parallel, if they are *keyed*, i.e., such as in the example above, can only be evaluated in a single, non-parallel task. The following example shows how to define a keyed window aggregation on a table. 
 
 <div class="codetabs" markdown="1">
 <div data-lang="java" markdown="1">
 {% highlight java %}
 Table table = input
-  .groupBy("a")
-  .window(GroupWindow w)  // define window
+  .window(window.as("w"))  // define window with alias
+  .groupBy("w, a") 
   .select("a, b.sum")     // aggregate
 {% endhighlight %}
 </div>
 
 <div data-lang="scala" markdown="1">
 {% highlight scala %}
 val table = input
-  .groupBy('a)
-  .window(w: GroupWindow) // define window
+  .window(window as 'w) // define window with alias
+  .groupBy('w, 'a)
   .select('a, 'b.sum)     // aggregate
 {% endhighlight %}
 </div>
 </div>
 
-The `GroupWindow` parameter defines how rows are mapped to windows. `GroupWindow` is not an interface that users can implement. Instead, the Table API provides a set of predefined `GroupWindow` classes with specific semantics, which are translated into underlying `DataStream` or `DataSet` operations. The supported window definitions are listed below. 
-By assigning the group-window an alias using `as`, properties such as the start and end timestamp of a time window can be accessed in the `select` statement.
+The `Window` parameter defines how rows are mapped to windows. `Window` is not an interface that users can implement. Instead, the Table API provides a set of predefined `Window` classes with specific semantics, which are translated into underlying `DataStream` or `DataSet` operations. The supported window definitions are listed below. Window properties such as the start and end timestamp of a time window can be accessed in the `select` statement.
```

Comment:
Window properties such as the start and end timestamp of a time window can be accessed in the `select` statement via window alias.